docs/kcl-lang/settings/project.md

```diff
@@ -45,6 +45,14 @@ This setting has the following nested options:
 
 
 
+**Default:** None
+
+
+##### enable_copilot
+
+Toggle copilot features
```

Comment:
Perhaps it doesn't matter as a hidden setting, but the description of others in this file read like full sentences. I'm not sure where this text is actually used, documentation perhaps? Ideally the content is synchronized with the settings UI.